File: 백준/Bronze/5622. 다이얼/다이얼.py
s = input()
sum = 0
c = 0

for i in s :
    c = ord(i)
    if c >= 87 :
        sum += 10
    elif c >= 84 :
        sum += 9
    elif c >= 80 :
        sum += 8
    elif c >= 77 :
        sum += 7
    elif c >= 74 :
        sum += 6  
    elif c >= 71 :
        sum += 5
    elif c >= 68 :
        sum += 4
    elif c >= 65 :
        sum += 3

print(sum)

File: 백준/Bronze/5622. 다이얼/다이얼.cc
#include <iostream>
#include <string>
using namespace std;

// 65~90 A~Z
int main(){
	string str;
	int sum = 0;
	cin >> str;
	
	for (char c : str) {
		if (c >= 'W') sum += 10;
		else if (c >= 'T') sum += 9;
		else if (c >= 'P') sum += 8;
		else if (c >= 'M') sum += 7;
		else if (c >= 'J') sum += 6;
		else if (c >= 'G') sum += 5;
		else if (c >= 'D') sum += 4;
		else if (c >= 'A') sum += 3;
	}
	
	cout << sum; 
}